src/components/PurchaseOrder/PODetails/PODetailsView.js

```diff
@@ -18,7 +18,7 @@ class DetailsView extends React.Component {
           <KeyValue label="Vendor" value={_.get(order, 'vendor_name')} />
         </Col>
         <Col xs={6}>
-          <KeyValue label="Created By" value={_.toString(_.get(order, 'created_by'))} />
+          <KeyValue label="Created By" value={_.get(order, 'created_by_name')} />
```

Comment:
It would be better to get rid of importing the whole Lodash lib and correct all used methods. What do you think?